tests/sarif_output.test.js

```diff
@@ -72,7 +72,11 @@ describe("SARIF", () => {
   it("debian", async () => {
     const sarif = await testSource(
       "localhost:5000/match-coverage/debian:latest",
+<<<<<<< HEAD
+      ["GHSA-9w8r-397f-prfh-Pygments", "GHSA-fp4w-jxhp-m23p-bundler"]
```

Comment:
Looks like this has a conflict

Reply:
yep, not resolved all things.
Give me some time.
I will convert into Draft and when all is merged reopen Pr.